src/main/java/org/example/C03_38_Palindrome_Number.java

```diff
@@ -0,0 +1,23 @@
+package org.example;
+
+import java.util.Scanner;
+
+public class C03_38_Palindrome_Number {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter your number: ");
+        int number = scanner.nextInt();
+        int reverse = 0;
+        int orig_number = number;
+
+        for (orig_number=number; number > 0; ){
```

Comment:
`orig_number=number` is not necessary. You already did that at line 11  `int orig_number = number;`.
Use `while` instead of `for`.

Reply:
done